requirements.txt

```diff
@@ -1,11 +1,49 @@
-docker
-glog
-numpy
-pika==0.11
-pydotplus
-pylint
-pymongo
-pytest
-pyyaml
-rouge
-six
+astroid==1.6.5
```

Comment:
Only need to include the top level package here. Please add `boto3` and `python_terraform` only (or a few extras). It would be easier to maintain.

Reply:
i actually did that initially. but right now, if you try to install a fresh boom copy using python 2, it will fail because internally some package is using an updated version of some other package which requires python3. i thought right now this problem has occurred only for this package but over time this might happen for other packages as well. so it is safest to specify all the package versions.

Reply:
Or only fix `boto3` and `python_terraform`? Or fix all packages in the original `requirements.txt`. It would be way easier to maintain.